source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -0,0 +1,394 @@
+      the App Services ``GroupID`` and ``AppID`` for the App you created above, 
+      to enable Tiered Sync Authentication:
+
+      .. code-block:: shell
+
+         curl --request POST --url https://realm.mongodb.com/api/admin/v3.0/groups/<groupID>/apps/<appID>/auth_providers -H "Content-Type: application/json" -H "Authorization: Bearer $TOKEN" --data "{\"Type\": \"tiered-sync-token\", \"Config\": {\"token\": \"$SECRET\""}}
+
+.. _tiered-sync-configure-and-start-server:
+
+Configure The EC2 Instance to Run the Tiered Sync Server
+--------------------------------------------------------
+
+Set up and run the Tiered Sync server on the EC2 instance.
+
+.. include:: /includes/important-stop-tiered-sync-server.rst
+
+.. procedure::
+
+   .. step:: Get the Tiered Sync Server Code
+
+      Use ``wget`` to get the Tiered Sync Server code:
+
+      .. code-block:: shell
+
+         wget https://tiered-sync-package.s3.amazonaws.com/d49a2f331f53cc0e6af450a067c4fd9184c3ae63-tiered_sync.tgz
+
+      Unzip the tar to get the files.
+
+      .. code-block:: shell
+
+         tar -xvf *.tgz
+
+      You now have a ``tiered_sync`` directory containing the server files.
+
+   .. step:: Complete the Tiered Sync Configuration Details
+
+      The ``tiered_sync`` directory contains a :file:`config.json` file you 
+      must edit with the appropriate values to configure the server.
+
+      .. code-block:: json
+
+         {
+           "clientAppId": "YOUR-APP-ID",
+           "query": {
+             "YOUR-COLLECTION-NAME": "truepredicate"
+           },
+           "cloudSyncServerAuth": "YOUR-AUTH-TOKEN",
+           "hostname": "YOUR-TIERED-SYNC-SERVER-DOMAIN"
+         }
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Field name
+           - Value
+
+         * - ``clientAppId``
+           - The App ID of the App Services App you created above.
+
+          
+         * - ``YOUR-COLLECTION-NAME``
+           - The name of the collection in your database where the Tiered 
+             Sync server should sync data. If you're using a 
+             template app, this is the ``Item`` collection.
+             Currently, ``truepredicate`` is the only supported query.
+
+         * - ``cloudSyncServerAuth``
+           - This is the Tiered Sync Server authorization token you generated
+             and saved as the ``SECRET`` in a step above.
+
+
+         * - ``hostname``
+           - The public DNS of your EC2 instance. This should resemble something
+             like ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``.
+
+   .. step:: Install Dependencies
+
+      The Tiered Sync server requires several dependencies, which are listed 
+      in the :file:`README.md` in the ``tiered_sync`` directory. Follow the 
+      instructions to install the required dependencies.
+
+   .. step:: Start the Tiered Sync Server
+
+      To start the server, from the ``tiered_sync`` directory:
+
+      .. code-block:: shell
+
+         sudo make up
```

Comment:
Do you know if the `sudo` is really required? I noticed in the demo that the SEs were using sudo a lot, it seemed random, but it should not be used without purpose, want to be sure we are using it correctly.

Reply:
Any time I was trying to do something with the tiered sync server code - i.e. run `make` - it was failing when I didn't use `sudo`. The instructions do say you may need to run from root, and I found that using sudo resolved errors I was having. I suspect we're running the server on a protected port, but that's just a guess - I am not actually sure what aspect of the server code required root access.

Reply:
That's because `sudo` in effect runs it as root.